pages/components/Collection/Collection.js

```diff
@@ -1,5 +1,69 @@
-const Collection = () => (
-  <h1>123</h1>
-)
+import S from "@/pages/components/Collection/Collection.module.scss"
+import React, { useState } from "react"
+import Image from "next/image"
+import prevItemImage from "../../../public/prevItem.png"
+import nextItemImage from "../../../public/nextItem.png"
+import { wines, wineType } from "@/pages/utils/varibles"
+import WineButton from "@/pages/components/WineButton"
+import CollectionItem from "@/pages/components/CollectionItem"
+import WineCollectionCircles from "@/pages/components/WineCollectionCircles"
+
+const Collection = () => {
+  const [ winesCollection, setWinesCollection ] = useState(wines)
+  const [ currentIndex, setCurrentIndex ] = useState(0)
+  const [ activeButton, setActiveButton ] = useState(null)
+
+  const sort = (event) => {
+    setCurrentIndex(0)
+    setWinesCollection(wines.filter((element) => {
+      return element.classification === event
+    }))
+  }
+
+  const changeColor = (index) => {
+    setActiveButton(index)
+  }
+
+  const nextItem = () => {
+    setCurrentIndex((prevIndex) => (prevIndex + 1) % winesCollection.length)
+  }
+
+  const prevItem = () => {
+    setCurrentIndex((prevIndex) => (prevIndex - 1 + winesCollection.length) % winesCollection.length)
+  }
+
+  return (
+    <>
+      <div id="container" className={ S.containerType }>
+        {
+          wineType.map((event, index) => {
+            const WineButtonText = activeButton === index ? ("/" + event.toUpperCase()) : event.toUpperCase()
```

Comment:
`const WineButtonText = activeButton === index ? ("/" + wineType.toUpperCase()) : wineType.toUpperCase()`

Here `activeButton === index` controls appearing of `/`. Duplication of `wineType.toUpperCase()` should lead you to this idea.


```js
const isActive = activeButton === index;
const prefix = isActive ? "/" : ""

const WineButtonText = prefix  + wineType.toUpperCase()
```